src/tty/windows/mod.rs

```diff
@@ -76,6 +76,20 @@ impl<'a> Pty<'a> {
     pub fn resize_handle(&self) -> impl OnResize + 'a {
         self.handle.clone()
     }
+
+    pub fn is_winpty(&self) -> bool {
+        match self.handle {
+            PtyHandle::Winpty(_) => true,
+            PtyHandle::Conpty(_) => false
+        }
+    }
+
+    pub fn is_conpty(&self) -> bool {
+        match self.handle {
+            PtyHandle::Winpty(_) => false,
+            PtyHandle::Conpty(_) => true
+        }
+    }
```

Comment:
It seems a bit pointless to me to have both `is_conpty` and `is_winpty` when one is just the inverse of the other.

I think just having `is_conpty` would be good, since that's currently the 'new feature' we want to check for. In the future if this is the commonly used thing, it might make more sense to switch this around to `is_winpty` to query for backwards-compatability support, but I wouldn't bother with that for now.

Reply:
I think that's reasonable. Will modify.